graylog2-server/src/main/java/org/graylog2/rest/models/system/ldap/requests/LdapSettingsRequest.java

```diff
@@ -89,6 +92,7 @@
     public static LdapSettingsRequest create(@JsonProperty("enabled") boolean enabled,
                                              @JsonProperty("system_username") @NotEmpty String systemUsername,
                                              @JsonProperty("system_password") @NotEmpty String systemPassword,
```

Comment:
I'm getting
```
There was an error fetching a resource: cannot PUT http://localhost:9000/api/system/ldap/settings (400). Additional information: Can not construct instance of org.graylog2.rest.models.system.ldap.requests.LdapSettingsRequest, problem: Null systemPassword
```
when trying to save an updated LDAP settings page with leaving the system password empty (i.e. to the "set" placeholder).

This change needs to allow for the `systemPassword` to be empty/null and check that the password has already beend set.